doc/rados/operations/balancer.rst

```diff
@@ -3,14 +3,15 @@
 Balancer
 ========
 
-The *balancer* can optimize the placement of PGs across OSDs in
-order to achieve a balanced distribution, either automatically or in a
-supervised fashion.
+The *balancer* can optimize the allocation of placement groups (PGs) across
```

Comment:
I think underscores fit our pattern better

Reply:
RST can't produce underscores. We have bold and italics.

Reply:
I meant the markup \_balancer\_